tests/integration/test_lists/test-db/l0_a10.yml

```diff
@@ -9,19 +9,52 @@ l0_a10:
       gpu:
       - '*a10*'
       linux_distribution_name: ubuntu*
+    terms:
+      stage: pre_merge
+      backend: pytorch
   tests:
   # ------------- PyTorch tests ---------------
   - disaggregated/test_disaggregated.py::test_disaggregated_single_gpu_with_mpirun[TinyLlama-1.1B-Chat-v1.0]
   - disaggregated/test_disaggregated.py::test_disaggregated_cuda_graph[TinyLlama-1.1B-Chat-v1.0]
   - disaggregated/test_disaggregated.py::test_disaggregated_mixed[TinyLlama-1.1B-Chat-v1.0]
   - disaggregated/test_disaggregated.py::test_disaggregated_overlap[TinyLlama-1.1B-Chat-v1.0]
+  - stress_test/stress_test.py::test_run_stress_test[TinyLlama-1.1B-Chat-v1.0_tp1-pytorch-stress-test]
+  - stress_test/stress_test.py::test_run_stress_test[TinyLlama-1.1B-Chat-v1.0_tp1-pytorch-stress-stage-alone]
+- condition:
+    ranges:
+      system_gpu_count:
+        gte: 1
+        lte: 1
+    wildcards:
+      gpu:
+      - '*a10*'
+      linux_distribution_name: ubuntu*
+    terms:
+      stage: pre_merge
+      backend: cpp
```

Comment:
I am not sure. Should this be "tensorrt"? or "cpp"? is "backend: cpp" is valid backend here in the yml file context?

Reply:
cpp backend  is not used in our stress test case.